source/main/zettlr.ts

```diff
@@ -840,4 +844,22 @@ export default class Zettlr {
   prompt (options: any): void {
     this._windowManager.prompt(options)
   }
+
+  /**
+   * Initialize the tray icon
+   *
+   * @return  {Tray}  tray class to manage the tray icon
+   */
+  initTray (): void {
+    let tempTray = new Tray('source/main/assets/icons/128x128.png') // Assign the icon image
+    // Define a context menu, triggered when right click the tray icon
+    const contextMenu = Menu.buildFromTemplate([
+      { label: 'Show Zettlr', type: 'normal' },
+      { label: 'Quit', type: 'normal' }
+    ])
+    tempTray.setToolTip('Zettlr editor') // A tooltip when move the cursor on the icon
+    tempTray.setContextMenu(contextMenu) // Set the context menu to tray icon
+    this.tray = tempTray
+    global.log.info('Successfully initialize the tray icon') //
```

Comment:
Logging is useful during development, but as this will end up as a pull request to Zettlr. I suggest remove it.

Reply:
I understand, I still need it to investigate where to call the initTray function could make the tray occur after the window displayed.